### PR TITLE
historic-blockhashes

### PR DESCRIPTION
**Pull request by zmitton**
I believe the contract implementation strategy is what held this up (EIP #210). Lets keep it simple. We won't have to worry about complex "stuffing" processes this way. And clients already have this data, so it should be relatively easy.

### CODE REVIEW

File: EIPS/eip-2427.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+lang: en
```
Comment (MicahZoltu) [inline EIPS/eip-2427.md:2]:
```suggestion
```
This isn't a valid EIP field.

File: EIPS/eip-2427.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+eip: 2427
+title: BLOCKHASH2
+author: Zac Mitton (@zmitton)
+status: DRAFT
+type: Standards Track
+category: Core
+created: 2019-12-04
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/242
+replaces: 210
```
Comment (MicahZoltu) [inline EIPS/eip-2427.md:11]:
```suggestion
```
Only final EIPs can be replaced.

File: EIPS/eip-2427.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
+---
+lang: en
+eip: 2427
+title: BLOCKHASH2
+author: Zac Mitton (@zmitton)
+status: DRAFT
+type: Standards Track
+category: Core
+created: 2019-12-04
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/242
```
Comment (MicahZoltu) [inline EIPS/eip-2427.md:10]:
```suggestion
discussions-to: https://github.com/ethereumclassic/ECIPs/issues/242
status: Draft
type: Standards Track
category: Core
created: 2019-12-04
```
Moves discussion-to up to match template for consistency.

**Review by MicahZoltu — CHANGES_REQUESTED** [review]
Along with the suggested changes, this EIP also needs all of the sections found in the template: https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md

Is there still interest in this over https://eips.ethereum.org/EIPS/eip-2935 ?  If not, it may be worthwhile to consolidate effort into one of the two.

**Comment by zmitton** [comment]
I posted this nearly a year ago after aggregating the best research from multiple other EIPs (themselves over 3 years old). Didn't receive so much as a single comment until now. Might as well just close this one because Vitalik has written one now which will probably actually be payed attention to.

**Comment by lightclient** [comment]
@zmitton do you have any feedback on how this process can be improved? It is really unfortunate this EIP received no traction when you authored it. Did you share this with anyone in the community (e.g. twitter, eth magicians, etc)?

**Comment by zmitton** [comment]
I guess my suggestion is that the person assigned to review it, does so. It was clear there was interest, so that means it should get moved to the next step of the process, or given a reason why not?

**Comment by MicahZoltu** [comment]
We have a had a problem historically of editors not proactively reading EIPs, which meant the only way to get them through the process was to find and pester an editor to read your EIP and review it.  We have gotten better and started to work through the backlog, but editors are still greatly understaffed so this generally continues to be a problem.

**Pull request closed without merging**